packages/atomic/src/components/search/atomic-generated-answer/typing-loader.tsx

```diff
@@ -1,9 +1,9 @@
 import {FunctionalComponent, h} from '@stencil/core';
 
 export const TypingLoader: FunctionalComponent = () => (
-  <div class="typing-indicator" aria-hidden="true">
-    <span></span>
-    <span></span>
-    <span></span>
+  <div part="container-background" class="typing-indicator" aria-hidden="true">
```

Comment:
"container" doesn't mean much here.
"typing-indicator" and "typing-indicator-dot"?
"typing-animation" and "typing-animation-dot"?

Reply:
Agree. Lets go for typing-animation